### ISSUE TITLE
Editor crashes when Back Space is pressed twice

### ISSUE THREAD

**Issue by rtcms** [curlpipe/ox#80]
**Describe the bug**
I am presented with this error message 

```
thread 'main' panicked at 'index out of bounds: the len is 0 but the index is 0', src/document.rs:385:21 
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace                                                                                     
Aborted (core dumped)
```

**To Reproduce**
1. Open ox editor via CLI
2. Press Back Space twice
3. You can see the error (ref screenshot) and the editor crashes

**Expected behavior**
Should not crash

**Screenshots**
![Screenshot from 2020-11-13 20-42-37](https://user-images.githubusercontent.com/35173068/99089104-e2a96e00-25f2-11eb-9f65-ad5a738cc794.png)

**Desktop (please complete the following information):**
 - OS: Linux 5.8.18-300.fc33.x86_64 (Fedora 33)

**Comment by curlpipe** [comment]
I've fixed this in 0.2.7 which will be released soon.

**Comment by rtcms** [comment]
Awesome! Thanks for the update
